t/01_style.t

```diff
@@ -12,4 +12,6 @@ ok system(qq{git grep -I -l '[#/ ]*SPDX-License-Identifier ' ':!t/01_style.t'})
 $out = qx{git grep -ne "check_var('ARCH',.*)" -e "check_var('BACKEND',.*)" ':!lib/Utils/Architectures.pm' ':!lib/Utils/Backends.pm' 'lib' 'tests'};
 ok $? != 0 && $out eq '', 'No check_var function to verify ARCH/BACKEND types' or diag $out;
 ok system(qq{git grep -I -l \\( -e "egrep" -e "fgrep" \\) ':!t/01_style.t' ':!CONTRIBUTING.md'}) != 0, 'No usage of the deprecated egrep and fgrep commands';
+$out = qx{git grep -I -l 'nots3cr3t' ':!data/wsl/Autounattend_*.xml' ':!data/yam/agama/auto/*plain_password.jsonnet' ':!data/yam/agama/auto/lvm_encrypted.jsonnet' 'data'};
```

Comment:
I asked developers about other plain password not in user or root, it will be very inefficient add a file to exclude here for each encryption scenario that we will have. You will have to exclude:
"luks2": { "password": "nots3cr3t" } and
"luks1": { "password": "nots3cr3t" } but now we have a way to pass it from the code like with AutoYaST, so we can avoid this.

Reply:
At here, we can only exclude file name, bug if every encryption scenario include 'encrypted' in the files' name, I can exclude them with ':!data/yam/agama/auto/*_encrypted.jsonnet',  Now I excluded all files which include plain password. If any update later, I can update this line again.

Reply:
Give us a time to pass the password in a different way. My suggestion is that you remove all the files in the exclusion list or at least the one in Yam and not merge this yet. The CI should not force name of files and the check will be removed if annoy other contributors that need to add a file each time they use encryption, a proper solution should be offered.

Reply:
Hello https://suse.slack.com/archives/C02CSAZLAR4/p1738922909487469 to add more context. 
You could use `$password` like https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/4c2b8c85c641ccde790353d4c8c1f5359bfbbb44/lib/services/users.pm#L126 to avoid hardcoding it.

Reply:
I cannot use '$password' or '$testapi::password' at here,  at CI phase, those variable is not defined.  
Report error message:
Use of uninitialized value $password in concatenation (.) or string at t/01_style.t line 17.